paypal/standard/models.py

```diff
@@ -240,6 +240,8 @@ class Meta:
     def __unicode__(self):
         if self.is_transaction():
             return self.format % ("Transaction", self.txn_id)
+        elif self.subscr_id:
```

Comment:
This should be `is_subscription()`, which should wrap the logic just like `is_transaction()`